python/langsmith/client.py

```diff
@@ -1442,7 +1471,7 @@ def multipart_ingest_runs(
             - The run objects MUST contain the dotted_order and trace_id fields
                 to be accepted by the API.
         """
-        if not create and not update:
+        if not create and not update and not feedback:
```

Comment:
very nit: slightly more readable if `not(create or update or feedback)`

Reply:
Updated